README.md

```diff
@@ -454,39 +454,75 @@ us from bridging this functionality into the React Native SDK code.
 #### Deep Linking
 
 [Deep Links](https://help.klaviyo.com/hc/en-us/articles/14750403974043) allow you to navigate to a particular
-page within your app in response to the user opening a notification. Familiarize yourself with the
-[React Native Guide](https://reactnative.dev/docs/linking) to deep linking, then read through the platform-specific
-instructions below.
+page within your app in response to the user opening a notification. This includes both custom URL schemes
+(e.g., `myapp://`) and Universal Links (iOS) / App Links (Android) which use standard HTTPS URLs.
 
-- [Android](https://github.com/klaviyo/klaviyo-android-sdk#Deep-Linking) instructions for handling intent filters
-- [iOS](https://github.com/klaviyo/klaviyo-swift-sdk#Deep-Linking)
-  As shown in the native SDK documentation, you can follow option 1 or 2.
+Klaviyo uses universal links for click tracking in emails and SMS messages. When a user clicks a Klaviyo
```

Comment:
nit: can you copy the text from the first iOS and android readmes here for consistency? 

>Klaviyo [Deep Links](https://help.klaviyo.com/hc/en-us/articles/14750403974043) allow you to navigate to a particular page within your app in response to the user opening a push notification, tapping on a link in an In-App Form, or by tapping on a Universal Link (aka Verified App Link on Android) from outside of the app. The Klaviyo Android SDK supports deep linking via custom URI schemes or Universal Links.

Reply:
note that we intentionally didn't say "SMS" here since we're launching without text channels